main.tf

```diff
@@ -40,6 +52,16 @@ resource "google_storage_bucket" "application" {
   }
 }
 
+# ------------------------------
+# Binding Google Tags to buckets
+# ------------------------------
+resource "google_tags_location_tag_binding" "binding" {
+  for_each  = { for bucket_tag in local.generated_map_bucket_tags : "${bucket_tag.bucket_name}.${bucket_tag.bucket_location}.${bucket_tag.tag_value}" => bucket_tag }
+  parent    = "//storage.googleapis.com/projects/_/buckets/${each.value.bucket_name}"
+  tag_value = "tagValues/tagBindings/{parent}/${each.value.tag_value}"
```

Comment:
Are you sure about this? If I remember correctly, the string should be something like `<tag key friendly name>/<tag value>`, and the tag key friendly name is something like `<numbers>/<key name>`.

I also think that would be better if the `tag_value` will be passed entirely from the outside.

Reply:
LGTM